### PR TITLE
refactored order details info

### LINKED ISSUE #83: Implement Order Details Component
Make an Order Details component that displays the full details of a specific order transaction.

- [x] Order transactions in ```OrderList.js``` must be clickable.
- [ ] Create a layout for order details.
- [ ] Implement state binds to display the full details of the order transaction.

### PR DESCRIPTION
**Pull request by BoJoNVi**
Fixes #83. Made an initial layout for the Order Tracker section. The design will be finalized soon.

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] screenshots/GIFs are attached :paperclip: in case of UI changes

### CODE REVIEW

**Comment by jcads** [comment]
I think `order tracker` is a bit misleading. Should we change it to `Order status`?

<br />
Also, don't be afraid to maximize the space. Put some padding under the section's heading text.

**Comment by jcads** [comment]
Check the data. The labels don't match the values. 
image

<br />

And maybe we can remove the `Time`?

**Comment by BoJoNVi** [comment]
Hey @jcads , in this PR I was just focusing on the Order Status.. Also, I've made another design for it.. Can you check which of the two is better?
image
image

**Comment by BoJoNVi** [comment]
> And maybe we can remove the `Time`?

I think we should not remove the time because I think it is important when dealing with transactions. But yes, I'll fix the values for that label.

**Comment by jcads** [comment]
> Hey @jcads, I've made another design for it.. Can you check which of the two is better?
> image
> image
> 
> But in my opinion, I prefer the first one.

For this one, let's just stick with the original design. 
image

Notice the size of the circle and how the spacing is utilized.

File: src/pages/orders/OrderDetails.js [hunk 1]
```diff
@@ -105,37 +119,63 @@ const OrderDetails = (props) => {
...
+                    <label>{order.createdAt.split("T")[0]}</label>
                   </div>
                   <div className="d-sm-flex flex-wrap">
                     <h3 className="mb-0 mr-2">Time: </h3>
-                    <label>{buyer.email}</label>
-                  </div>
-                  <div className="d-sm-flex flex-wrap">
-                    <h3 className="mb-0 mr-2">Total: </h3>
-                    <label>{order.orderAmount * order.quantity}</label>
+                    <label>{order.createdAt.split("T")[1].split(".")[0]}</label>
```
Comment (jcads) [inline src/pages/orders/OrderDetails.js:126]: Using `split`s in dealing with dates is inefficient. Use the built-in [`Date`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date) API instead. 

<br />

And format of the date should be in `mm/dd/yyyy`.

**Comment by BoJoNVi** [comment]
I've now implemented what you've suggested @jcads 😄. 

I'll try to add the horizontal line for the Order Status now.
image

**Comment by jcads** [comment]
> I've now implemented what you've suggested @jcads smile.
> 
> I'll try to add the horizontal line for the Order Status now.
> image

Again. Refer on the Figma prototype for the design and to get the actual proportions of the components.

File: src/pages/orders/OrderDetails.js [hunk 2]
```diff
@@ -86,56 +102,93 @@ const OrderDetails = (props) => {
...
                   <div className="d-sm-flex flex-wrap">
                     <h3 className="mb-0 mr-2">Time: </h3>
-                    <label>{buyer.email}</label>
-                  </div>
-                  <div className="d-sm-flex flex-wrap">
-                    <h3 className="mb-0 mr-2">Total: </h3>
-                    <label>{order.orderAmount * order.quantity}</label>
+                    <label>
+                      {dateTime.getHours()}:{dateTime.getMinutes()}:
+                      {dateTime.getSeconds()}
```
Comment (jcads) [inline src/pages/orders/OrderDetails.js:134]: Nice! We can go a step further. Try exploring the docs. _hint: each one just uses a method from the Date class_

You could also put these in helper functions to tidy it out a bit.

**Pull request merged**